packages/esm-patient-list-app/src/api/api-remote.ts

```diff
@@ -105,14 +105,24 @@ export async function getPatientListMembers(cohortUuid: string, ac = new AbortCo
     throw error;
   }
 
+  const currentDate = new Date();
   const searchQuery = results.map((p) => p.patient.uuid).join(',');
   const result = await openmrsFetch(`/ws/fhir2/R4/Patient/_search?_id=${searchQuery}`, {
     method: 'POST',
     signal: ac.signal,
   });
 
   const patients: Array<PatientListMember> = result.data.entry.map((e) => e.resource);
-  return patients;
+  const validPatients = patients.filter((patient) => {
```

Comment:
I think simply doing `  const validPatients = patients.filter((patient) => !patient.endDate);`

Reply:
It's not though, because it's possible to add a cohort member with an expiration date, so a cohort member might have an end date but still _currently_ be part of the cohort (there's no UI for this yet; it doesn't mean the feature doesn't exist).

Reply:
gotcha! Thanks @ibacher